vignettes/new-application.Rmd

```diff
@@ -285,6 +286,21 @@ add_ui_sidebar_basic(list(s1, s2), append = FALSE)
 add_ui_sidebar_basic(list(s1, s2), append = FALSE, tabname = "Options")
 ```
 
+### program/ui_sidebar_right.R  
+
+Create UI components for the right sidebar and register them with the framework using
+a call to *add_ui_sidebar_right()*.  
+
+```{r, eval=F}
+# -- Create UI sidebar Elements
+s1 <- rightSidebarTabContent(id = 1, icon = "desktop", title = "Tab 1 - Plots", active = TRUE, 
```

Comment:
Let's align the inputs vertically like in the examples below for body1/body2, it makes it easier for a user to follow.